app/router.js

```diff
@@ -151,6 +151,11 @@ router.map(function() {
       });
       this.route('translations');
     });
+    this.route('event-settings', function() {
```

Comment:
Move these to content

```
/admin/content/events/topics
/admin/content/events/sub-topics
/admin/content/events/types
```

Reply:
Or even better, you could put all three into just a single page too `/admin/content/events` since the functionality needed isn't complex.

Reply:
Ok, I will try doing that. This means that form for creating all three things would be in same page? Currently I display them in same page but for new creation everything has different form.

Reply:
Thanks @niranjan94 I will fix this. :smile: